lib/pattern.js

```diff
@@ -2176,7 +2176,8 @@
   var dataPrefix = "data-".concat(vl.ns),
```

Comment:
zijn dat dan wijzigingen die wij in AIV code doen, of die wij nu nog ergens overnemen van hun?

Reply:
Wijzigingen die wij in de AIV code doen. @Pieter-Beckers heeft hetzelfde moeten doen voor de UUID validatie/pattern.

Reply:
@pascalriquier de bedoeling is dat wijzigingen van ons aan de lib centraal ook gebeurd zodat we de lib op termijn kunnen verwijderen. Dat proces volg ik op, maar is voorlopig te traag om de wijzigingen niet reeds bij ons in de lib uit te voeren.